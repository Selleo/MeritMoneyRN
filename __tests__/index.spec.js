import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'

import Index from '../index.js'

it('renders correctly', () => {
  const tree = renderer.create(<Index />).toJSON()

  expect(tree).toMatchSnapshot()
})
