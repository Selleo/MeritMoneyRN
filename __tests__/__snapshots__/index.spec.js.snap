// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#222f3f",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignSelf": "center",
          "opacity": 0,
          "position": "absolute",
          "top": 25,
          "transform": Array [
            Object {
              "scale": 0.3,
            },
          ],
          "zIndex": 2,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "borderColor": "#30B97E",
            "borderRadius": 45,
            "borderWidth": 5,
            "height": 90,
            "justifyContent": "center",
            "width": 90,
            "zIndex": 3,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "backgroundColor": "#ffffff",
              },
              Object {
                "borderRadius": 40,
                "height": 80,
                "width": 80,
              },
            ]
          }
        >
          <Image
            source={
              Object {
                "uri": "https://i.imgur.com/ckqCXMS.png",
              }
            }
            style={
              Array [
                Object {
                  "borderRadius": 40,
                  "height": 80,
                  "width": 80,
                },
                Object {
                  "zIndex": -1,
                },
              ]
            }
          />
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <ActivityIndicator
        animating={true}
        color="#999999"
        hidesWhenStopped={true}
        size="small"
      />
    </View>
  </View>
</View>
`;
